app/models/concerns/foreman_remote_execution/errors_flattener.rb

```diff
@@ -6,7 +6,8 @@ def flattened_validation_exception
 
     def flattened_errors
       errors = Hash.new { |h, k| h[k] = [] }
-      self.errors.each_key do |key|
+      # rubocop:disable Performance/HashEachMethods
```

Comment:
Please put here a comment about the reason why we can't do that (`ActiveModel::Errors` missing `each_key` method)